docs/modules/ROOT/pages/whats-new.adoc

```diff
@@ -4,50 +4,58 @@
+A new CP Map data structure is the latest addition to our xref:cp-subsystem:cp-subsystem.adoc[CP Subsystem]. Unlike traditional maps built for availability (AP), CP Maps guarantee data consistency even during network partitions, ensuring that every member within the cluster sees the same data. CP Maps are ideal for storing sensitive information or managing mission-critical workflows.
 
 The CP Subsystem is the Hazelcast component that builds a strongly consistent layer for a set of distributed data structures. As well as network partitions, the CP Subsystem withstands server and client failures. xref:management-center:cp-subsystem:dashboard.adoc[Hazelcast Management Center] allows you to inspect the CP Subsystem as a whole – for individual data record access, you can use the Hazelcast Command Line Client (CLC).
 
-For more information on how to use this new data structure, see xref:data-structures:cpmap.adoc[].
+For more information on how to use this new `CPMap` data structure, see xref:data-structures:cpmap.adoc[].
 
-== Thread-Per-Core (TPC) now Generally Available [.enterprise]*Enterprise* 
+== Thread-Per-Core (TPC) now generally available (GA)
+ [.enterprise]*Enterprise* 
 
 Enable Thread-Per-Core (TPC) on both clients and cluster members to improve system performance, especially in environments where the number of CPU cores on your machines does not match the recommended hardware. TPC is an alternative to traditional Staged Event-Driven Architecture (SEDA) that supports easier scaling based on the number of available cores.
 
 For more information on how to enable and configure TPC, see xref:cluster-performance:thread-per-core-tpc.adoc[].
 
-== Compute Isolation with User Code Namespaces [.enterprise]*Enterprise* 
+== Manage resources effectively with User Code Namespaces
+[.enterprise]*Enterprise* 
 
-User Code Namespaces provide a container for Java classpath resources such as user code, and accompanying artifacts such as property files. This allows you to effectively manage access to resources in different namespaces through either static or dynamic configuration. This is an Enterprise feature that extends and replaces the now deprecated xref:clusters:legacy-ucd.adoc[User Code Deployment] functionality.
+User Code Namespaces provide a container for Java classpath resources, such as user code and accompanying artifacts such as property files. This enables you to effectively manage access to resources in different namespaces through either static or dynamic configuration. This is an Enterprise feature that extends and replaces the now deprecated xref:clusters:legacy-ucd.adoc[User Code Deployment] functionality.
```

Comment:
Just saying:
https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/e/enable-enables
https://developers.google.com/style/word-list#enable
https://www.plainenglish.co.uk/the-a-z-of-alternative-words.html#E

I note that Google and MS style guides do not like allows either, so perhaps "this means that you can effectively...."

;-)

I feel this is especially important on this page where enables is used both to mean allows and to mean turn on

Reply:
lol ok, maybe there's at third way out this allows/enables hell